docs/architecture/adt-014-proportional-slashing.md

```diff
@@ -0,0 +1,87 @@
+# ADR 14: Proportional Slashing
+
+## Changelog
+
+- 2019-10-15: Initial draft
+
+## Context
+
+In Proof of Stake-based chains, centralization of consensus power amongst a small set of validators causes harm to the network due to increased risk of censorship, liveness failure, fork attacks, etc.  However, while this centralization causes a negative externality to the network, it is not directly felt by the stakers contributing towards delegating towards already large validators.  We would like a way to pass on the negative externality cost of centralization onto those large validators and their stakers.
+
+## Decision
+
+### Design
+
+To solve this problem, we will implement a procedure called Proportional Slashing.  The desire is that the larger a validator is, the more they should be slashed.  The first naive attempt is to make a validator's slash percent proportional to their share of consensus voting power.
+
+```
+slash_amount = power // power is the faulting validator's voting power.
```

Comment:
Typically, this is `k * power`, where `k` is some on-chain constant.

Reply:
I moved the mention of the constant multiplier further down for simplicity.